packages/composites/anchor/src/endpoint/price/bluna.ts

```diff
@@ -1,41 +1,29 @@
-import { AdapterContext, AdapterRequest, AdapterResponse } from '@chainlink/types'
-import { Config } from '../../config'
-import * as view from '@chainlink/terra-view-function-adapter'
+import { AdapterContext, AdapterRequest } from '@chainlink/types'
+import { Config, FIXED_POINT_DECIMALS } from '../../config'
 import { Validator } from '@chainlink/ea-bootstrap'
+import { BigNumber, ethers } from 'ethers'
```

Comment:
We are both importing `BigNumber` and `ethers` here, but in the code we're referencing BigNumber as both `BigNumber` and `ethers.BigNumber`. Suggest being consistent with this.